src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/use-a-template.mdx

```diff
@@ -24,50 +24,47 @@ Templates are pre-designed and reusable configurations that help to streamline t
 
 ## Getting the template
 
-[Access Azion Console](https://console.azion.com/) and click the **+ Create** button on the homepage.
+To get a template:
 
-This will open a modal where you can select the **Templates** option and choose the one you want to start with.
+1. [Access Azion Console](https://console.azion.com/).
+2. Click the **+ Create** button on the homepage.
+3. In the modal, select the Templates option.
+4. Browse and choose the one you want to start with.
 
 :::note
-Some templates are integrated with third-party services and you'll need to comply certain requirements, such a creating an account, generating tokens, or setting up configurations on the third-party platform. Go to the [guides page](https://www.azion.com/en/documentation/products/guides/#azion-templates) and check more details on the specific template you're willing to use.
+Some templates are integrated with third-party services and you'll need to comply certain requirements, such a creating an account, generating tokens, or setting up configurations. Additionally, it could be necessary to activate some Azion products. Go to the [guides page](https://www.azion.com/en/documentation/products/guides/#azion-templates) and check more details on the specific template you're willing to use.
```

Comment:
```suggestion
Some templates are integrated with third-party services and you'll need to comply with certain requirements, such as creating an account, generating tokens, or setting up configurations. Additionally, it could be necessary to activate some Azion products. Go to the [guides page](https://www.azion.com/en/documentation/products/guides/#azion-templates) and check more details on the specific template you're willing to use.
```